store/sqlstore/post_store.go

```diff
@@ -6,18 +6,17 @@ package sqlstore
 import (
 	"database/sql"
 	"fmt"
-	"net/http"
-	"regexp"
-	"strconv"
-	"strings"
-	"sync"
-
 	sq "github.com/Masterminds/squirrel"
 	"github.com/mattermost/mattermost-server/v5/einterfaces"
 	"github.com/mattermost/mattermost-server/v5/mlog"
 	"github.com/mattermost/mattermost-server/v5/model"
 	"github.com/mattermost/mattermost-server/v5/store"
 	"github.com/mattermost/mattermost-server/v5/utils"
+	"net/http"
+	"regexp"
+	"strconv"
+	"strings"
+	"sync"
```

Comment:
These imports should go in the first block, with the rest of the standard library imports

Reply:
Thanks @mgdelacroix.
I have reverted the import statement changes [here](https://github.com/mattermost/mattermost-server/pull/14884/commits/11c082b2bb48a58773d588d94215f024588d8188)